analyzers/tests/SonarAnalyzer.UnitTest/Helpers/HashCodeTest.cs

```diff
@@ -56,9 +58,77 @@ public void DictionaryContentHash_StableForImmutableDictionary()
             var numbers = Enumerable.Range(1, 1000);
             var dict1 = numbers.ToImmutableDictionary(x => x, x => x);
             var dict2 = numbers.OrderByDescending(x => x).ToImmutableDictionary(x => x, x => x);
-            var hashCode1 = SonarAnalyzer.Helpers.HashCode.DictionaryContentHash(dict1);
-            var hashCode2 = SonarAnalyzer.Helpers.HashCode.DictionaryContentHash(dict2);
+            var hashCode1 = HashCode.DictionaryContentHash(dict1);
+            var hashCode2 = HashCode.DictionaryContentHash(dict2);
             hashCode1.Should().Be(hashCode2);
         }
+
+        [TestMethod]
+        public void EnumerableUnorderedContentHash_Empty()
+        {
+#pragma warning disable CA1825 // Avoid zero-length array allocations
```

Comment:
I'd be fine with disabling this at the beginning of this file once. And not enabling it.